Section_I/law14.tex

```diff
@@ -42,6 +42,13 @@
 \item must remain on \added{or behind} his goal line facing the kicker in the physical competition. In the virtual competition, it may be facing any direction. In any case, it needs to stay between the goalposts until the ball has been kicked
 \end{itemize}
 
+\added{The Kicker:
+	
+	\begin{itemize}
+		\item must be placed inside the field of play
+		\item must be placed behind the penalty mark and at least one robot length away from the mark
```

Comment:
maybe better: "must be placed behind and at least one robot length away from the penalty mark"